examples/public_private_flow_logs/main.tf

```diff
@@ -0,0 +1,28 @@
+module "vpc" {
+  source = "../.."
```

Comment:
Ditto other- replace with tf registry source, add a README, and recommend adding ">= 0.0.1" version